src/source/Ice/IceAgent.c

```diff
@@ -1757,6 +1757,15 @@ STATUS iceAgentInitRelayCandidates(PIceAgent pIceAgent)
     return retStatus;
 }
 
+
+STATUS trunStateFailedFn(UINT64 data, PSocketConnection pSocketConnection) {
```

Comment:
It's a good idea to include the pSocketConnection even if it's not used right now. Two recommendations:

1) Make this as the first parameter
2) Add UNUSED_PARAM(pSocketConnection) to fix unused param warning